ui/dashboard: add DLQ metrics, reorder the charts

dt opened the pull request.

dt committed:
ui/dashboards: add retry queue size, dlq reasons

Release note: none.
Epic: none.

dt committed:
ui/dashboards: reorder LDR dash

Lag is the top-line number we care about (literally); the events applied
or bytes moved to apply them are secondary. Accordingly this moves the
max lag metric top, followed by the latency metric, and then the count
and byte metrics.

While I'm here, reformat the file.

Release note: none.
Epic: none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/126854)
<!-- Reviewable:end -->

kev-cao commented:
Could I see a screenshot of this chart with some sample data?

dt commented:
<img width="1034" alt="Screenshot 2024-07-11 at 7 40 56 AM" src="https://github.com/cockroachdb/cockroach/assets/15615/e6b11cc2-f703-4fc0-97f1-6264b1cee80d">

dt committed:
crosscluster/logical: limit bytes per server, not proc

The server is what will OOM.

Release note: none.
Epic: none.

dt committed:
crosscluster/logical: mark levels closed even without checkpoints

If a new level is started, the prior is closed even it if did not get a checkpoint.

Release note: none.
Epic: none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/126854)
<!-- Reviewable:end -->
